pom.xml

```diff
@@ -543,6 +543,18 @@
         <version>3.2.1</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.msgpack</groupId>
```

Comment:
So you decided to keep these? That's cool, just misunderstood what you said before, but no worries.